server/gitrest/packages/gitrest-base/src/utils/filesystems.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import fs from "node:fs";
-import { Redis as IoRedis, RedisOptions as IoRedisOptions } from "ioredis";
+import { Redis as IoRedis, RedisOptions as IoRedisOptions, type Cluster } from "ioredis";
```

Comment:
Educate me, why there is a `type` here but others not?

Reply:
For this one, I think I was getting a compiler error, with the recommended change being to add the "type".
But not sure why this is needed here and not in the others.